Add Ruby/Rails and Mediawiki files (#148)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/rubyonrails.md

---
name: Ruby/Rails
category: Languages and Framework
description: Ruby on Rails (RoR) is a popular web framework written in the Ruby programming language.
download_url: https://github.com/rails/rails/tags
works_on_arm: true
supported_minimum_version:
    version_number: 3.1.1
    release_date: 07/08/2011

optional_info:
    homepage_url: https://rubyonrails.org/
    support_caveats:
    alternative_options:
    getting_started_resources:
        arm_content:  
        partner_content: 
        official_docs: https://github.com/rails/rails/blob/main/README.md
    arm_recommended_minimum_version:
        version_number:
        release_date:

optional_hidden_info:
    release_notes__supported_minimum:
    release_notes__recommended_minimum:

>     other_info: There are no release notes or binaries present for arm64. 3.1.1 version of rails is downloaded by using gems and checked.

Author: ```suggestion
    other_info: There are no release notes or binaries present for arm64. Version 3.1.1 of rails was installed and tested using gems.
```
